tests/flow/test_distinct.py

```diff
@@ -76,7 +76,7 @@ def test_distinct_with_order(self):
         result = graph1.query("MATCH (p:PARENT)-[:HAS]->(c:CHILD) RETURN DISTINCT p.name ORDER BY c.name")
         self.env.assertEqual(result.result_set, [['Stevie'], ['Mike'], ['James']])
 
-        result = graph1.query("UNWIND range(0,3) AS a UNWIND range(4,7) AS b RETURN DISTINCT a ORDER BY b")
+        result = graph1.query("UNWIND range(0,3) AS a UNWIND range(4,7) AS b WITH DISTINCT a ORDER BY b RETURN a ORDER BY a DESC")
```

Comment:
why this change is needed?

Reply:
The original order by sorted `b` values which are all the same (4), there's no definition for sorting values that are equal, as such no one can make any assumption on the order.